src-tool/Pact/Analyze/Eval/Core.hs

```diff
@@ -251,24 +251,40 @@ evalCore (StrContains needle haystack) = do
 -- TODO (RS): we should add warnings to the stack, allowing
 --            to return shims in case, the content is not statically known
 evalCore (StrHash sT) = eval sT <&> unliteralS >>= \case
-  Nothing -> throwErrorNoLoc notStaticErr
+  Nothing -> do
+    let h = "HsVo-gcG-pk1BciGr2xovMyR7sVH0Kt9gTgqicXDXMM"
```

Comment:
What do these constants mean? How were they determined?

Reply:
I'm confused by this too, is the literal what you get from hashing "hello pact"? Why would we want this hash in the case where the input can't be `unliteralS`'d?